masterlist.yaml

```diff
@@ -15082,6 +15091,11 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'Winterhold Restored & WR JKs Skyrim Patch' ]
         condition: 'active("Winterhold Restored.esp") and active("Winterhold Restored - JKs Skyrim Patch.esp") and not active("OCW_WR2_JK_FEPatch.esp")'
+      - <<: *patch3rdParty
+        subs: 
+          - 'Immersive College of Winterhold'
+          - '[Ultimate College of Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/44797)'
+        condition: 'active("CollegeOfWinterholdImmersive.esp") and not active("UltimateCollege.esp")'
```

Comment:
Once is probably enough to show the patch message, I'd say leave this once and remove the other.